src/main/java/com/danielagapov/spawn/Controllers/OAuthController.java

```diff
@@ -30,6 +32,16 @@ public ResponseEntity<AbstractUserDTO> googleSignIn(@AuthenticationPrincipal OAu
         }
     }
 
+    //
+    @GetMapping("/sign-in?sub=sub")
+    public ResponseEntity<Map<Boolean, FullUserDTO>> signIn(@RequestParam("sub") String sub) {
```

Comment:
```suggestion
    public ResponseEntity<FullUserDTO> signIn(@RequestParam("sub") String sub) {
```
If you go with your idea

Reply:
Implemented [here](https://github.com/Daggerpov/Spawn-App-Back-End/pull/113/commits/0d55bf93c1e77c24b94f462956280c39e65b8cba)